README.md

```diff
@@ -13,16 +13,22 @@ layout:
     visible: true
 ---
 
-# Kontrol
-
 <div data-full-width="true">
 
 <figure><img src=".gitbook/assets/kontrol logo yellow.png" alt=""><figcaption></figcaption></figure>
 
 </div>
 
-[**Kontrol**](https://github.com/runtimeverification/kontrol) combines **KEVM** and [Foundry](https://book.getfoundry.sh/) to grant developers the ability to perform [formal verification](https://en.wikipedia.org/wiki/Formal\_verification) without learning a new language or tool. This is especially useful for those who are not verification engineers. Additionally, developers can leverage Foundry test suites they have already developed and use symbolic execution to increase the level of confidence.
```

Comment:
I think we should add back in the mentions of Foundry, as the key UX point of Kontrol is that it does integrate with Foundry, so it does'nt take developers learning any new syntax or techniques above what they already know. But perhaps we can de-emphasize it and move it to the bullet-points below. I don't think we need to mention KEVM so prominently.

Reply:
Addressed by @Stevengre